Binary files gnucash-1.6.5.orig/.configure.in.swp and gnucash-1.6.5/.configure.in.swp differ
diff -urN gnucash-1.6.5.orig/configure.in gnucash-1.6.5/configure.in
--- gnucash-1.6.5.orig/configure.in	Tue Mar  5 23:50:14 2002
+++ gnucash-1.6.5/configure.in	Tue Mar  5 23:58:20 2002
@@ -373,9 +373,9 @@
   )
  )
 else
- AC_CHECK_LIB(db, dbopen, DB_LIBS="-ldb",
-  AC_CHECK_LIB(db1, dbopen, DB_LIBS="-ldb1",
-   AC_CHECK_LIB(db-3, __db185_open, DB_LIB="-ldb-3",
+ AC_CHECK_LIB(db-3, __db185_open, DB_LIBS="-ldb-3",
+  AC_CHECK_LIB(db, dbopen, DB_LIBS="-ldb",
+   AC_CHECK_LIB(db1, dbopen, DB_LIBS="-ldb1",
     AC_MSG_ERROR([Your db library is missing db 1.85 compatibility mode])
    )
   )
